.buildkite/test-pipeline.yaml

```
@@ -1348,6 +1348,21 @@ steps:
     - pytest -v -s tests/distributed/test_nccl_symm_mem_allreduce.py
     - pytest -v -s tests/v1/distributed/test_dbo.py
 
+- label: Elastic EP Scaling Test
+  timeout_in_minutes: 20
+  gpu: b200
+  optional: true
+  working_dir: "/vllm-workspace/"
+  num_gpus: 4
```

@libertyeagle Any reason it needs to be b200? I think we could remove the gpu requirement like this:
```suggestion
  optional: true
  working_dir: "/vllm-workspace/"
  num_gpus: 4
```